### PR TITLE
Add quotes to sliders v2

### PR DESCRIPTION
**Pull request by Thynix**
Reopened version of #21 that's merged with next. It also reduces vertical space around the carousel. Deployed to testing.freenetproject.org.

### CODE REVIEW

File: pages/index.py [hunk 1]
```diff
@@ -27,35 +27,59 @@ def slider_item(title, text, active=""):
...
+""") + "\n\n" + """<span class="hidden-xs">""" + _("""
+*'Daddy, where were you when they took freedom of the press away from the Internet?'*
+— Mike Godwin ([read more](about.html))
+""") + """</span>""", "active"),
             slider_item(_("Host a Website"), _("""
 Need a website nobody can take over? That is hosted for free? That is very
 resistant to attacks? Publish it on Freenet!
-""")),
+""") + "\n\n" + """<span class="visible-xs">""" + _("""
+*'I never had to worry no more about security of my sites.'*
```
Comment (Thynix) [inline pages/index.py]: Emotive though it may be, I'm skeptical about using quotes with grammar errors, especially in such a prominent position. Thoughts?

**Comment by bertm** [comment]
I'm not really convinced by the scattered availability of user quotes on
mobile and the lack of attribution there. Spacing looks good though. I'll
have another look once I get home, which will be tomorrow or on sunday.

**Comment by ArneBab** [comment]
@bertm did you check how it looks? The mobile format is quite limiting, but I think it turned out well.

File: pages/index.py (hunk 1 above)
Comment (ArneBab) [inline pages/index.py]: I’m not sure; I like it … but I’m no native speaker, so grammar does not hurt as much for me. Maybe use the other quote for visible-xs instead?

**Comment by bertm** [comment]
It does look better on desktop, yes.

Then again, I'm not entirely convinced by the minor design details, but let's keep nitpicking for later (read: for when I have a real suggestion for improvement). The improvements so far are ready enough.

**Comment by Thynix** [comment]
Deployed.

**Pull request merged**